internal/integration/spec.go

```diff
@@ -47,6 +48,14 @@ type MetricDef struct {
 	Name string `yaml:"provider_name"`
 }
 
+type entityNameProps struct {
```

Comment:
can you add just some info regarding this struct?
I understand why is there but in the future it might be confusing